.github/actions/install-dependencies/action.yml

```diff
@@ -33,7 +33,7 @@ runs:
     - uses: actions/setup-go@v5
       if: ${{ inputs.skip_go == 'false' }}
       with:
-        go-version: '1.20'
+        go-version: '1.22'
```

Comment:
Stefan and me were discussing this way of hardcoding versions in a https://github.com/zeta-chain/protocol-contracts/pull/212#discussion_r1674318232

To make this more dynamic we could have a .versions file in every one of our repositories, centralizing the versioning in just one place.

Reply:
I think having a central shared `actions/setup-go` that sets the version once would be reasonable.

You could read this out of the go.mod file directly rather than a separate file.